lib/gui/models/selection-state.js

```diff
@@ -16,338 +16,325 @@
-  this.hasImage = () => {
-    return Boolean(this.getImage());
-  };
+/**
+ * @summary Remove drive
+ * @function
+ * @public
+ *
+ * @example
+ * selectionState.removeDrive();
+ */
+module.exports.removeDrive = () => {
+  Store.dispatch({
+    type: Store.Actions.REMOVE_DRIVE
+  });
+};
 
-  /**
-   * @summary Remove drive
-   * @function
-   * @public
-   *
-   * @example
-   * SelectionStateModel.removeDrive();
-   */
-  this.removeDrive = () => {
-    Store.dispatch({
-      type: Store.Actions.REMOVE_DRIVE
-    });
-  };
+/**
+ * @summary Remove image
+ * @function
+ * @public
+ *
+ * @example
+ * selectionState.removeImage();
+ */
+module.exports.removeImage = () => {
+  Store.dispatch({
+    type: Store.Actions.REMOVE_IMAGE
+  });
+};
 
-  /**
-   * @summary Remove image
-   * @function
-   * @public
-   *
-   * @example
-   * SelectionStateModel.removeImage();
-   */
-  this.removeImage = () => {
+/**
+ * @summary Clear selections
+ * @function
+ * @public
+ *
+ * @param {Object} options - options
+ * @param {Boolean} [options.preserveImage] - preserve image
+ *
+ * @example
+ * selectionState.clear();
+ *
+ * @example
+ * selectionState.clear({ preserveImage: true });
+ */
+module.exports.clear = (options = {}) => {
+  if (!options.preserveImage) {
     Store.dispatch({
       type: Store.Actions.REMOVE_IMAGE
     });
-  };
-
-  /**
-   * @summary Clear selections
-   * @function
-   * @public
-   *
-   * @param {Object} options - options
-   * @param {Boolean} [options.preserveImage] - preserve image
-   *
-   * @example
-   * SelectionStateModel.clear();
-   *
-   * @example
-   * SelectionStateModel.clear({ preserveImage: true });
-   */
-  this.clear = (options = {}) => {
-    if (!options.preserveImage) {
-      Store.dispatch({
-        type: Store.Actions.REMOVE_IMAGE
-      });
-    }
-
-    Store.dispatch({
-      type: Store.Actions.REMOVE_DRIVE
-    });
-  };
+  }
 
-  /**
-   * @summary Check if a drive is the current drive
-   * @function
-   * @public
-   *
-   * @param {String} drive - drive device
-   * @returns {Boolean} whether the drive is the current drive
-   *
-   * @example
-   * if (SelectionStateModel.isCurrentDrive('/dev/sdb')) {
-   *   console.log('This is the current drive!');
-   * }
-   */
-  this.isCurrentDrive = (drive) => {
-    if (!drive) {
-      return false;
-    }
+  Store.dispatch({
+    type: Store.Actions.REMOVE_DRIVE
+  });
+};
 
-    return drive === _.get(this.getDrive(), [ 'device' ]);
-  };
-
-});
+/**
+ * @summary Check if a drive is the current drive
+ * @function
+ * @public
+ *
+ * @param {String} drive - drive device
+ * @returns {Boolean} whether the drive is the current drive
+ *
+ * @example
+ * if (selectionState.isCurrentDrive('/dev/sdb')) {
+ *   console.log('This is the current drive!');
+ * }
+ */
+module.exports.isCurrentDrive = (drive) => {
+  if (!drive) {
+    return false;
+  }
 
-module.exports = MODULE_NAME;
+  return drive === _.get(this.getDrive(), [ 'device' ]);
+};
```

Comment:
Kind of annoying that the GH diff-display can't do a better job with this file. Ho hum :-/